content/attacks/posts/2022-08-01-Nomad.md

```diff
@@ -0,0 +1,31 @@
+An investigation led by samczsun, the head of security at Paradigm, a web3 investment firm, revealed that one of Nomad's smart contracts had been updated in a way that made it vulnerable to transaction spoofing. This vulnerability enabled users to transfer funds between blockchains without proper verification of the transaction amounts. As a result, some users were able to withdraw funds that did not belong to them. Blockchain audit company Zellic reached the same conclusion in its own analysis.
+In response to the hack, Nomad took the initiative to offer a bounty of up to 10% for the return of looted funds. Those who returned more than 90% of the stolen funds would be considered "white hat" hackers and would be allowed to keep the remaining amount. Nomad made it clear that no legal action would be taken against individuals who returned the funds. As a result, white hat hackers returned over $36 million to the platform, contributing to the recovery efforts.
+
+## Attackers
+
+The true identity of the initial attacker remains unknown, as the hack itself was a result of an error. However, it is known that a significant number of accounts exploited this vulnerability. Once other attackers became aware of the situation, they mobilized large numbers of bots to execute similar attacks.
+
+## Losses
+
+After a frenzied hack from hundreds of wallets, the bridge’s TVL dropped from $190,740,000 to $1,794 in mere hours.
+
+## Timeline
+
+- **August 1, 2022 11:25 PM GMT:** [The project team reported](https://twitter.com/nomadxyz_/status/1554246853348036608) that they know about the hack.
+- **August 2, 2022 10:26 PM GMT:** [The project team stated](https://twitter.com/nomadxyz_/status/1554413278406721537?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1554413278406721537%7Ctwgr%5Ee9a62d942e20b470f1122ce321dad21cece3434b%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.theverge.com%2F2022%2F8%2F2%2F23288785%2Fnomad-bridge-200-million-chaotic-hack-smart-contract-cryptocurrency) that they informed law enforcement authorities and engaged reputable firms specializing in blockchain intelligence and forensics.
+- **August 3, 2022 04:05 AM GMT:**[Nomad announced an up to 10% bounty:](https://twitter.com/nomadxyz_/status/1554679735006859264) those who return >90% of funds.
+- **August 5, 2022:** [Nomad published a detailed analysis](https://medium.com/nomad-xyz-blog/nomad-bridge-hack-root-cause-analysis-875ad2e5aacd) of the root causes of the attack.
+
+## Security Failure Causes
+
+**Smart Contract Vulnerability:** The security failure in the Nomad bridge occurred due to a mistake made during the contract update process. This highlights the need for comprehensive code review and change management practices to identify and address vulnerabilities within the smart contract code. Conducting thorough audits can help detect overlapping functionalities or potential vulnerabilities that could be exploited by attackers.
```

Comment:
Remove last two sentences and add more details about the vulnerability itself

Reply:
Corrected